File: uStock-Development-Android/app/src/main/java/data_structures/SpamReport.kt
package data_structures

import java.util.Date

data class SpamReport(
    val reporter: String,
    val spamContent: String,
    val reportedUser: String,
    val reportedPost: String,
    val reportDate: Date?,
    val resolutionDate: Date?,
    val status: String
)

File: uStock-Development-Android/taskThree_models/Data Structures/SpamReport.swift
//
//  SpamReport.swift
//  uStockIOS-V2-AI
//
//  Created by Gal Cohavy on 6/29/23.
//

import Foundation

public struct SpamReport: Codable {
    let reporter: String
    let spamContent: String
    let reportedUser: String
    let reportedPost: String
    let reportDate: Date?
    let resolutionDate: Date?
    let status: String
    
    enum CodingKeys: String, CodingKey {
        case reporter, spamContent, reportedUser, reportedPost, reportDate, resolutionDate, status
    }
}
